freud/__init__.py

```diff
@@ -1,8 +1,8 @@
 # Copyright (c) 2010-2024 The Regents of the University of Michigan
 # This file is from the freud project, released under the BSD 3-Clause License.
 
-# density,; diffraction,; environment,; order,
-from . import box, cluster, data, interface, locality, msd, parallel, pmft
+# cluster,; density,; diffraction,; environment,; interface,; msd,;
+from . import box, data, interface, locality, order, parallel, pmft
```

Comment:
Am I looking at the latest version? I thought we fixed the missing `cluster` and `msd` imports yesterday?

Reply:
Looks like I forgot to push, sorry about that!